app/services/phone_sms_sender.rb

```diff
@@ -16,7 +16,7 @@ def initialize(message, from_user_id, to_mobile_phone, media_url, force_opt_out)
   end
 
   def send!
-    RadicalRetry.perform_request do
+    RadicalRetry.perform_request(raise_original: true) do
```

Comment:
we retry the lookups twice, maybe we should limit this too? failures to send still cost $ im assuming

Reply:
nah, it's ok - the lookups can get hit a lot more with entering phone numbers -  volume on sending SMS will be a fraction of that

Reply:
nah, it's ok - the lookups can get hit a lot more with entering phone numbers -  volume on sending SMS will be a fraction of that